language/cloud-client/v1/snippets.py

```diff
@@ -24,17 +24,20 @@
 import argparse
 import sys
 
-from google.cloud import language
-from google.cloud.language import enums
-from google.cloud.language import types
-import six
 
-
-# [START language_sentiment_text]
 def sentiment_text(text):
+    # [START language_sentiment_text]
+    import six
+    from google.cloud import language
+    from google.cloud.language import enums
+    from google.cloud.language import types
+
     """Detects sentiment in the text."""
     client = language.LanguageServiceClient()
 
+    # TODO(developer): Uncomment the following line to run this code.
+    # text = 'Your text to analyze, e.g. Hello, world!'
```

Comment:
It would be better to remove the parameter and have the actual text defined here. This will reduce confusion for the user. Only have commented out variables when absolutely necessary (ex. a project ID).

Reply:
So, the function takes no input parameters as text, and we uncomment text to something like this:
text = 'Hello World'

Yeah?

Reply:
And what about the other functions that have a gcs_uri input parameter?

Reply:
Yes, you are correct on the 'Hello World' example. For the gcs_uri parameter, upload the sample file to the cloud-samples-data storage bucket, and then update to where you uploaded it. For example: `gcs_uri = 'gs://cloud-samples-data/language/your-sample-file.txt'`

Reply:
Did we finalize that new sample rubric yet?

Reply:
This page the example page alix? https://cloud.google.com/natural-language/docs/quickstart-client-libraries

Reply:
It's not officially finalized because we're still waiting on UX testing for the script vs. function formatting. I believe eliminating the use of commented out variables when possible was non-controversial, though.

I don't think that link is the "official" example page, though those look like good samples to me.